src/main/java/com/arpnetworking/metrics/impl/TsdMetricsFactory.java

```diff
@@ -107,37 +205,66 @@ protected TsdMetricsFactory(final Builder builder) {
     public static class Builder {
 
         /**
+         * Public constructor.
+         */
+        public Builder() {
```

Comment:
If the serviceName and clusterName are required, wouldn't it be better to have them as part of the Builder constructor?